.github/workflows/ci.yml

```
@@ -1,43 +1,53 @@
-name: Tests
+name: CI
+
+env:
+  protoc_version: '3.x'
+
 on:
   push: {branches: [master]}
-  pull_request: {branches: [master]}
+  pull_request:
 
 jobs:
-  analyze:
-    name: "Static analysis"
-
+  static_analysis:
+    name: Static analysis
     runs-on: ubuntu-latest
+
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-node@v1
+    - uses: actions/setup-node@v2
       with:
         node-version: 15.x
         check-latest: true
     - uses: arduino/setup-protoc@v1
-      with: {version: 3.x}
+      with: {
+        version: "${{env.protoc_version}}",
+        repo-token: "${{github.token}}"
+      }
```

```suggestion
      with:
        version: "${{env.protoc_version}}",
        repo-token: "${{github.token}}"
```

YAML only requires curly braces for single-line maps. Same below.